### PR TITLE
[merged] We started enumerating in public, and you won't believe what he saw

### PR DESCRIPTION
**Pull request by cgwalters**
See https://bugzilla.gnome.org/show_bug.cgi?id=764131

### CODE REVIEW

**Comment by cgwalters** [comment]
@cgwalters-bot delegate=gatispaeglis

File: src/libostree/ostree-dummy-enumtypes.c [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+ * License along with this library; if not, write to the
+ * Free Software Foundation, Inc., 59 Temple Place - Suite 330,
+ * Boston, MA 02111-1307, USA.
+ */
+
+#include "ostree-dummy-enumtypes.h"
+
+/* Exported for backwards compat - see 
+ * https://bugzilla.gnome.org/show_bug.cgi?id=764131
+ * https://github.com/ostreedev/ostree/pull/294
```
Comment (gatispaeglis) [inline src/libostree/ostree-dummy-enumtypes.c]: Probably no need for a link to pull request here? as it does not contain any relevant info.

File: src/libostree/ostree-dummy-enumtypes.c [hunk 2]
```diff
@@ -0,0 +1,32 @@
+/* This file declares a stub function that is only exported
+ * to pacify ABI checkers - no one could really have used it.
+ *
+ * Copyright (C) 2015 Red Hat, Inc.
```
Comment (gatispaeglis) [inline src/libostree/ostree-dummy-enumtypes.c]: Wrong year, if that matters? :)

**Comment by cgwalters** [comment]
Fixup :arrow_up:

**Comment by gatispaeglis** [comment]
Don't really know what this enum generation is, but the build works fine now.

@cgwalters-bot r+ 963f5fa

**Pull request closed without merging**